src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -248,6 +248,8 @@ namespace System.Drawing
 
         public static Color Purple => new Color(KnownColor.Purple);
 
+        public static Color RebeccaPurple => new Color(KnownColor.RebeccaPurple);
```

Comment:
nit: Would likely be beneficial to move this after MenuHighlight so it matches the enum order.

Reply:
sure

Reply:
there is no menuhighlight in this file. Should i move it to the end of yellow green or keep it as it is.

Reply:
There is no MenuHighlight here. Should i move it to the end of yellowgreen or keep it as it is?

Reply:
I'd move it to after yellow green. @safern or I can log a bug to ensure the missing ones get added.

Reply:
done